src/routes/safe/components/Apps/trackAppUsageCount.ts

```diff
@@ -0,0 +1,35 @@
+import { SafeApp } from 'src/routes/safe/components/Apps/types'
+import local from 'src/utils/storage/local'
+
+export const APPS_DASHBOARD = 'APPS_DASHBOARD'
+
+export type AppTrackData = {
+  [safeAppId: string]: {
+    timestamp: number
+    openCount: number
+    txCount: number
+  }
+}
+
+export const countOpen = (app: SafeApp): void => {
+  const trackData = local.getItem<AppTrackData>(APPS_DASHBOARD) || {}
+  let currentOpenCount = trackData[app.id]?.openCount
+  local.setItem(APPS_DASHBOARD, {
+    ...trackData,
+    [app.id]: {
+      ...trackData[app.id],
+      timestamp: Date.now(),
+      openCount: currentOpenCount ? ++currentOpenCount : 1,
+      txCount: trackData[app.id]?.txCount || 0,
```

Comment:
This is not needed, it will be written from `...trackData[app.id]`.

Reply:
I added this logic because the first time the user opens the app we want to initiate the `txCount` to 0 and the property does not exist yet

Reply:
Oh this goes with the other comment! Will change it as per suggested 👍